DiscordChatExporter.Gui/ViewModels/Components/DashboardViewModel.cs

```diff
@@ -283,6 +283,12 @@ await Parallel.ForEachAsync(
 
                         Interlocked.Increment(ref successfulExportCount);
                     }
+                    catch (ChannelNotExportedException ex) {
+                        _snackbarManager.Notify(ex.Message.TrimEnd('.'));
+
+                        // FIXME: not exactly successful, but not a failure either. Not ideal to duplicate the line
+                        Interlocked.Increment(ref successfulExportCount);
```

Comment:
This could also be handled in line 284, thus not needing to duplicate the line, but then error handling would be spread around inside of at the obvious place. Neither is 100% clean.

Reply:
Can you explain what duplication you're referring to?

Reply:
If you click into the diff link, you'll see the original line at 284 which I'm duplicating at 289. Slightly iffy but I can't find a cleaner way to express this logic.